src/components/table/table.tsx

```diff
@@ -135,7 +135,7 @@ export const Table: FC<TableComponentProps> = ({
           >
             {selectable && (
               <div className={cx('table-cell', 'checkbox-cell')}>
-                {(isAnyChecked || hoveredRow === index) && (
+                {(selectedRowIds?.length > 0 || hoveredRow === index) && (
```

Comment:
Please clarify the purpose of the change here and above. Why `isAnyChecked ` does not suit us?

Reply:
As discussed with Teo, when you have entered to bulk change mode, checkboxes should be visible at another pages, isAnyChecked variable related to current visible data only